rbd/encryption.go

```diff
@@ -0,0 +1,116 @@
+// +build !octopus,!nautilus
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// /* force XSI-complaint strerror_r() */
+// #define _POSIX_C_SOURCE 200112L
+// #undef _GNU_SOURCE
+// #include <errno.h>
+// #include <stdlib.h>
+// #include <rados/librados.h>
+// #include <rbd/librbd.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+// EncryptionFormat: encryption format type
+type EncryptionFormat C.uint
+
+// Possible values for EncryptionFormat:
+// RBD_ENCRYPTION_FORMAT_LUKS1: LUKS v1
+// RBD_ENCRYPTION_FORMAT_LUKS2: LUKS v2
+const (
+	RBD_ENCRYPTION_FORMAT_LUKS1 = EncryptionFormat(C.RBD_ENCRYPTION_FORMAT_LUKS1)
```

Comment:
Please consider using regular Go naming, like `EncryptionFormatLUKS1`, no need to prefix `rbd`, as that is the name of the package.

Reply:
I agree with @nixpanic wholeheartedly. I thought that the `revive` tool in our CI may also point out this problem, but even if it doesn't this change should be made.

Reply:
I thought it might make sense to keep the same librbd and go-ceph constant names.  But since you both disagree, I defer to your decision.